lib/web/mage/adminhtml/wysiwyg/tiny_mce/tinymce4Adapter.js

```diff
@@ -375,19 +375,21 @@ define([
          */
         openFileBrowser: function (o) {
             var typeTitle = this.translate('Select Images'),
-                storeId = this.config['store_id'] !== null ? this.config['store_id'] : 0,
+                storeId = this.config['store_id'] ? this.config['store_id'] : 0,
                 frameDialog = jQuery('div.mce-container[role="dialog"]'),
-                self = this,
                 wUrl = this.config['files_browser_window_url'] +
                     'target_element_id/' + this.getId() + '/' +
                     'store/' + storeId + '/';
 
+		this.initialize(this.getId(), this.config);
```

Comment:
We will call `initialize` 3 times, it's a very strange approach, why we need to do it?